ng-dev/pr/merge/strategies/autosquash-merge.ts

```diff
@@ -122,3 +118,12 @@ export class AutosquashMergeStrategy extends MergeStrategy {
     return null;
   }
 }
+
+/** Gets the absolute file path to the commit-message filter script. */
+function getCommitMessageFilterScriptPath(): string {
+  // We resolve the script using module resolution as in the package output
+  // the worker might be bundled but exposed through a subpath export mapping.
+  return require.resolve(
```

Comment:
Will this work within our own repository with `ts-node`?

Reply:
In our repo it would find the `.js` output built from Bazel. It would also still work with `ts-node` though.

Reply:
Actually, I now understand what you were referring to. Let me check.

Reply:
Fixed with the latest commit.